core/spoofax.compiler/src/main/resources/mb/spoofax/compiler/adapter/editor_services/ResolveMultiTaskDef.java.mustache

```diff
@@ -0,0 +1,164 @@
+package {{baseResolveMultiTaskDef.packageId}};
+
+import mb.aterm.common.TermToString;
+import mb.common.editor.ResolveResult;
+import mb.common.option.Option;
+import mb.common.region.Region;
+import mb.common.result.Result;
+import mb.common.util.ListView;
+import mb.constraint.pie.ConstraintAnalyzeTaskDef;
+import mb.jsglr.common.JsglrParseException;
+import mb.jsglr.common.TermTracer;
+import mb.pie.api.ExecContext;
+import mb.pie.api.None;
+import mb.pie.api.OutTransient;
+import mb.pie.api.Supplier;
+import mb.pie.api.TaskDef;
+import mb.resource.ResourceKey;
+import mb.resource.hierarchical.ResourcePath;
+import mb.stratego.common.StrategoException;
+import mb.stratego.common.StrategoRuntime;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.spoofax.interpreter.terms.IStrategoList;
+import org.spoofax.interpreter.terms.IStrategoTerm;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@{{adapterProject.scope.qualifiedId}}
+public class {{baseResolveMultiTaskDef.id}} implements TaskDef<{{baseResolveMultiTaskDef.id}}.Args, Option<ResolveResult>> {
```

Comment:
As discussed, the `ResolveTaskDef` and `ResolveMultiTaskDef` should be merged into one.